requirements/env_common_3_8.yml

```diff
@@ -27,3 +27,4 @@ dependencies:
     - xgboost
     - gensim  # used by mol2vec
     - tensorboard
+    - basis-set-exchange
```

Comment:
This should not be a required dependency for anyone

Reply:
I was removing it but saw that it was used in many places, and removing it will require adding try except everywhere.
So i have shifted this dependency from common to torch env.